### ISSUE TITLE
custom data

### ISSUE THREAD

**Issue by sandipgiri576** [jla-gardner/graph-pes#96]
I  want to test the PaiNN model with my  own custom data. Is the dataset format similar to the ASE db like in schnetpack library.? Could you provide a relevent config.yaml file for this. I have enegy, force and coordinate information. And another issue, I have found, during HPC SLURM run. It needs a interactive wandb mode, which is quite fine during collab notebook run, but for slurm it seems issue. Is there a way off this feature in yaml file?  Thank you.

**Comment by jla-gardner** [comment]
Hi there @sandipgiri576 - thanks for wanting to use `graph-pes`, and for raising these questions 😄 

Let me try to answer your questions:

## Training dataset format

The easiest way (for now) to provide `graph-pes-train` with a dataset of structures is to do the following:

```yaml
data:
    train:
        +file_dataset:
            path: path/to/train.xyz
            cutoff: 3.7
    valid:
        +file_dataset:
            path: path/to/valid.xyz
            cutoff: 3.7
```

where `train.xyz` and `valid.xyz` are text files containing your ASE atoms objects (see [here](https://wiki.fysik.dtu.dk/ase/ase/io/io.html) for more details). So for now, you would need to write a script that converts an ASE `.db` file into an `.xyz` one.

I am in the process of implementing a way to use ASE `.db` files directly in `graph-pes`, and will let you know when that is ready (hopefully later today!)

## Weights and Biases with SLURM

Thank you for raising this point! I have tried to address it in the docs [here](https://jla-gardner.github.io/graph-pes/cli/graph-pes-train.html#weights-and-biases) - please let me know if this isn't clear 😄 

TLDR; you can turn `wandb` off by using 

```yaml
wandb: null
```

in your config, or you can avoid the interactive step by logging into W&B before submitting your HPC job.

**Comment by sandipgiri576** [comment]
great. I see this wandb part. Although there are some changes in schnetpack painn hyperparameter and your graph-pes painn hyperparameter, I am going to check how it performs.

**Comment by sandipgiri576** [comment]
can you store all the  metric data and other information in a josn file? metrics.csv file not properly allign.
